rust_src/src/cmds.rs

```diff
@@ -58,7 +58,7 @@ fn move_point(n: LispObject, forward: bool) -> () {
 ///
 /// Depending on the bidirectional context, the movement may be to the
 /// right or to the left on the screen.  This is in contrast with
-/// \\[right-char], which see.
+/// [right-char], which see.
```

Comment:
This isn't right. `\\[foo]` means that describe-function will show the keybinding for the interactive function foo. `[foo]` has no special meaning.